docs/operator-docs/collection-storage-best-practices.md

```diff
@@ -0,0 +1,191 @@
+---
+id: collection-storage-best-practices
+title: Collection Storage Best Practices
+hide_title: true
+---
+
+# Collection Storage Best Practices
```

Comment:
```suggestion
# Collection Cost Tracking Best Practices
```

Wondering if this would be a better title to emphasize the cost tracking focus of this advice.

Reply:
I combined them